### DISCUSSION TITLE
Is react-native-video supported for the web?

### DISCUSSION THREAD

**Discussion by ShaimaaSabry** [TheWidlarzGroup/react-native-video#4137] [Q&A] [answered]
### What are you trying to do?

I am using react-native-video and it is not running on the web. I want to know whether the library supports the web, and if so how to fix the error I am getting. If not, I want to know the alternative for playing DRM videos on react-native web.

### What have you tried?

Running react-native-video on the web, which fails with the error below.

### Relevant code

```
Uncaught TypeError: (0 , react_native_1.requireNativeComponent) is not a function
    at AppEntry.bundle?platform=web&dev=true&hot=false&lazy=true&transform.engine=hermes&transform.routerRoot=app:99585:63
```

### Platforms

Web

**Comment by freeboub** [comment, marked as answer]
No not yet. There is an open PR here: https://github.com/TheWidlarzGroup/react-native-video/pull/3958
To be followed in linked ticket
